<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExprBuilder</class>
 <widget class="QMainWindow" name="ExprBuilder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Builder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="expression">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>expression</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="operators">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>104</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="values">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>104</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>55</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>operators</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>55</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>values</string>
    </property>
   </widget>
   <widget class="QPushButton" name="compButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>comp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eqButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>eq</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tenButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hundredButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="DropZone" name="result" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">#result {
background-color: rgb(255, 255, 255);
}</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="showValue">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Show value</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="showAll">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>82</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>show all</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="freeText">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>82</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>free</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>410</y>
      <width>76</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>new</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="multipleSelection">
    <property name="geometry">
     <rect>
      <x>134</x>
      <y>190</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>multiple</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="lhsFreeText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>free</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="showText">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>show as</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="extras">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>251</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>extra:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>319</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DropZone</class>
   <extends>QWidget</extends>
   <header>dropzone.hh</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
